syntax = "proto3";


service TensorSender {
	// Sends a Tensor
	rpc SendTensor(TensorRequest) returns (TensorReply) {}
}

message TensorRequest {
	string tensor_name = 1;
}

message TensorReply {
	string tensor_str = 1;
}
